tools/data-api/internal/endpoints/routing.go

```diff
@@ -0,0 +1,37 @@
+package endpoints
+
+import (
+	"github.com/go-chi/chi/v5"
+	"github.com/hashicorp/pandora/tools/data-api/internal/endpoints/infrastructure"
+	"github.com/hashicorp/pandora/tools/data-api/internal/endpoints/v1"
+	"github.com/hashicorp/pandora/tools/data-api/internal/repositories"
+)
+
+func Router(router chi.Router) {
+	router.Route("/infrastructure", infrastructure.Router)
+	router.Route("/v1/microsoft-graph/beta", func(r chi.Router) {
+		opts := v1.Options{
+			ServiceType:     repositories.MicrosoftGraphV1BetaServiceType,
+			UriPrefix:       "/v1/microsoft-graph/beta",
+			UsesCommonTypes: false,
```

Comment:
Graph uses CommonTypes fwiw:

```suggestion
			UsesCommonTypes: true,
```